src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -461,7 +462,7 @@ private void advanceBlock() throws IOException {
             while (blockIterator != null && blockIterator.hasNext()) {
                 final Block block = blockIterator.next();
                 if (block.getSize() > 0) {
-                    final int bufferSize = Math.min(2000000, block.getSize() > 100000000 ? 10000000 : (int) block.getSize());
+                    final int bufferSize = Math.min(2_000_000, block.getSize() > 100_000_000 ? 10_000_000 : (int) block.getSize());
```

Comment:
Is this a bug here?  I totally assumed those were the same number.

Reply:
On closer inspection, this whole line seems busted.  
It seems like what they meant to write was `Math.min(2_000_000, Math.min(10_000_000, block.getSize()))`, but then typoed the number of 0's.  But I think as it's currently written (or as I think they intended it, it just reduces to `Math.min(2_000_000, block.getSize())` anyway.  

I'm not sure what the correct behavior is, but this seems wrong in any case.

Reply:
I just added '_' so it (human)readable....if you want to change the numbers, please use a followup PR.

Reply:
I know, it just surprised me that those numbers weren't the same, which made me look at it.